test/cql-pytest/custom_decorators.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from test_config import IS_SCYLLA_VERSION
+
+
+# The "scylla_only" decorator can be used by tests for Scylla-only features,
+# which do not exist on Apache Cassandra. A test using this decorator will be
+# skipped if running with "run-cassandra".
+scylla_only = pytest.mark.skipif(condition=not IS_SCYLLA_VERSION,
+                                 reason="The test can run only with Scylla version")
+
+# "cassandra_bug" is similar to "scylla_only", except instead of skipping
+# the test, it is expected to fail (xfail) on Cassandra. It should be used
+# in rare cases where we consider Scylla's behavior to be the correct one,
+cassandra_bug = pytest.mark.xfail(condition=not IS_SCYLLA_VERSION,
+                                  reason="The test expected to fail")
```

Comment:
The reason "The test expected to fail" doesn't explain anything on an xfail :-)
The message we have now in cassandra_bug() is "A known Cassandra bug". What's wrong with that?